### ISSUE TITLE
Maximum call stack size exceeded

### ISSUE THREAD

**Issue by legion49f** [WebexSamples/hookbuster#8]
I have built hookbuster into a containerized application and have had the container crash with this trace back a few times. 

failed to execute Logger#warn RangeError: Maximum call stack size exceeded
hookbuster_1  |     at walkAndFilter (/node_modules/@webex/plugin-logger/dist/logger.js:92:23)
hookbuster_1  |     at walkAndFilter (/node_modules/@webex/plugin-logger/dist/logger.js:122:23)

Any help will be appreciated. I am not a JS developer unfortunately but i have some inclination that this might be due to a recursion loop.

Thanks in advance!

**Comment by jbenyovs** [comment]
@legion49f not sure how to reproduce this, would you mind describing the exact steps? Generally "Maximum call stack size exceeded" would happen if you are calling a function over and over again until the call stack size is reached. So yes, like a recursion loop, that cannot stop for some reason. I don't think that the hookbuster app has any recursions built in, but if you have the exact steps and infrastructure needed to reproduce this, I might take a closer look.

**Comment by legion49f** [comment]
Sure, we have built hookbuster into a container image which is now merged to master and this happens when the container image is left running for over a period of time. The steps to reproduce are just to leave hookbuster running for some time. It's worth pointing out that this might be an issue with the WebEx logger plugging imported by hookbuster and not an issue with the code from this application.
